### PR TITLE
feat(access_rule): add support for managed_challange

### PR DESCRIPTION
**Pull request by duraje**
same as  previous PR for rate limit

### CODE REVIEW

File: .changelog/1457.txt [hunk 1]
```diff
@@ -0,0 +1,6 @@
+```release:enhancement
+resource/cloudflare_access_rule: add support for managed_challenge action
+```
+
+https://github.com/cloudflare/terraform-provider-cloudflare/issues/1456
+https://github.com/cloudflare/terraform-provider-cloudflare/pull/1457
```
Comment (jacobbednarz) [inline .changelog/1457.txt:6]:
```suggestion
```

**Comment by jacobbednarz** [comment]
API references: https://api.cloudflare.com/#account-level-firewall-access-rule-create-access-rule

**Review by jacobbednarz — APPROVED** [review]

**Comment by jacobbednarz** [comment]
thanks for this @duraje 👏

**Comment by duraje** [comment]
you are welcome @jacobbednarz. With this in place i think we will be the first customer who fully replaced legacy_captcha with managed_captcha in automated fashion. I have already manually switched the two remaining access_rules and just waiting to add it back in terraform. Thank you for your support

**Comment by jacobbednarz** [comment]
that is awesome news and congrats on the move! hopefully the provider is adding value and making managing the configuration a bit easier for you and your org.

**Comment by duraje** [comment]
for us the Cloudflare terraform provider it is core of managing our Cloudflare account ;-)

**Pull request merged**